tools/pot/openvino/tools/pot/api/helpers.py

```diff
@@ -0,0 +1,253 @@
+# Copyright (C) 2020-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+from xmlrpc.client import boolean
+
+from openvino.tools.pot import Metric, DataLoader, IEEngine, \
+    load_model, save_model, compress_model_weights, create_pipeline
+
+class QuantizationParameters(object):
+    """
+    Basic parameters of the quantization method.   
+    """
+    def __init__(self):
+        """
+        Initializes quantization parameters with some reasonable verified defaults.
+        """
+        self._model_name = "model"
+        self._target_device = "ANY"
+        self._method_name = "DefaultQuantization"
+        self._preset = "performance"
+        self._subset_size = 300
+        self._model_type = None
+        self._fast_bias_correction = True
```

Comment:
It would be nice if these properties can be set in the `__init__` method, so you can set them in one line, e.g. `parameters = QuantizationParameters(model_name="awesome_model",  preset="mixed")`. I do like having these sane defaults with nothing required.

Reply:
Good suggestion!

Reply:
Agree with @helena-intel